#!/usr/bin/env python
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("line_tolerance", double_t, 0, "Line Tolerance for RANSAC", 0.3, 0.0, 1.0)
gen.add("max_view_distance", double_t, 0, "Max View Distance", 5.0, 0.0, 20.0)

exit(gen.generate( "auv_navigation", "slalom_processor", "Slalom"))
